week06_policy_based/reinforce_pytorch.ipynb

```diff
@@ -20,8 +20,11 @@
     "import sys, os\n",
     "if 'google.colab' in sys.modules and not os.path.exists('.setup_complete'):\n",
     "    !wget -q https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/setup_colab.sh -O- | bash\n",
-    "    !pip install -q gym[classic_control]==0.18.0\n",
+    "    !pip install -q gym[classic_control]==0.26.0\n",
     "    !touch .setup_complete\n",
```

Comment:
Could you put this line after all of your changes?

Reply:
Sure, fixed it.